metadata.yaml

```diff
@@ -10,21 +10,26 @@ description: |-
   converted to liquid runoff with no heat transfer.
   Spin up starts 1 Jan 1958 and runs to 1 Jan 2019.
 notes: |-
-  Source code: https://github.com/COSIMA/access-om2
-  License: Attribution-NonCommercial-NoDerivatives 4.0 International (http://creativecommons.org/licenses/by-nc-nd/4.0/legalcode)
-  Conditions of use:
-  We request that users of this or other ACCESS-OM2 model code or output data:
+  COSIMA request that users of this or other ACCESS-OM2 model code or output data:
   (a) consider citing Kiss et al. (2020) [http://doi.org/10.5194/gmd-13-401-2020]
   (b) include an acknowledgement such as the following:
   "The authors thank the Consortium for Ocean-Sea Ice Modelling in Australia (COSIMA; http://www.cosima.org.au)
   for making the ACCESS-OM2 suite of models available at https://github.com/COSIMA/access-om2.
   Model runs were undertaken with the assistance of resources from the National Computational Infrastructure (NCI),
   which is supported by the Australian Government."
-  (c) let us know of any publications which use these models or data so we can add them to our list:
+  (c) let COSIMA know of any publications which use these models or data so they can add them to their list:
   https://scholar.google.com/citations?hl=en&user=inVqu_4AAAAJ
 keywords:
-  - one
-  - cosima
-  - access-om2-1
-  - jra55-do-v1.4.0
-  - iaf
+- JRA55
+- access-om2
+- interannual
+- global
+realm:
+- ocean
+- seaIce
+nominal_resolution: 100 km
+reference: https://doi.org/10.5194/gmd-13-401-2020
+license: CC-BY-4.0
+model: access-om2
+version: 1.0
```

Comment:
Should we be setting this on the other releases also?

Reply:
I was trying to get around some logic errors that @CodeGat has now fixed in order to check later stages of the CI.

The intention of the design was that devs wouldn't need to set the version, there are tools to do this, but these initial cases found some logic bugs/omissions.

So short answer, no, it shouldn't be required.